modules/cluster-logging-deploy-cli.adoc

```diff
@@ -12,6 +12,11 @@ You can use the {product-title} CLI to install the Elasticsearch and Cluster Log
 * Ensure that you have the necessary persistent storage for Elasticsearch. Note that each Elasticsearch node
 requires its own storage volume.
 +
+[NOTE]
+====
+If you are using a local volume for persistent storage, do not use a raw block volume (`volumeMode: block` in the `LocalVolume` object) because Elasticsearch cannot use raw block volumes.
```

Comment:
```suggestion
If you use a local volume for persistent storage, do not use a raw block volume, which is described with `volumeMode: block` in the `LocalVolume` object. Elasticsearch cannot use raw block volumes.
```

> Avoid using parentheses in general text because text in parentheses is decreased in importance.